cfpb/cfgov-refresh
# Deprecate Wagtail welcome page

This page can currently only be seen by those users with Django admin permission (developers and admins). Those users don't always get sent to the page upon logging in, but only when logging in with certain `?next=/something/` parameters in the login URL.

Besides that, the content and design of this page has some issues, so we've decided to remove it. This PR does so.

See platform#3800 for background.

## How to test this PR

1. Login and logout and confirm that the normal flow works correctly.
2. When logged out, try to visit a page like http://localhost:8000/admin/template_debug/jobmanager/job_listing_list/. You should get redirected to http://localhost:8000/login/?next=/admin/template_debug/jobmanager/job_listing_list/, and then, upon logging in, go to the right page.
3. When logged out, try to visit a page like http://localhost:8000/admin/pages/319/edit/. You should get redirected to http://localhost:8000/login/?next=/admin/pages/319/edit/, and then, upon logging in, get redirected to http://localhost:8000/admin/.  <--  this is the case affected by this PR! Previously this is how you would end up at /login/welcome/. This is actually a bug, I think, as there's no reason why we shouldn't get to the right page, but I am deliberately not addressing that in this PR.

## Checklist

_[Feel free to delete any checkboxes that are not applicable to this PR.]_

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing with the feature/section you're addressing, e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"
